v0/destinations/vero/data/VeroTrackConfig.json

```
@@ -0,0 +1,19 @@
+[
+  {
+    "sourceKeys": "userId",
+    "destKey": "identity.id",
+    "sourceFromGenericMap": true
+  },
+  {
+    "sourceKeys": "event",
+    "destKey": "event_name"
+  },
+  {
+    "sourceKeys": "properties",
+    "destKey": "data"
+  },
+  {
+    "sourceKeys": "timestamp",
+    "destKey": "extras.check_id"
```

```suggestion
    "destKey": "extras.check_id",
     "sourceFromGenericMap": true
```
We should verify the timestamp we are providing is accepted by them or not.

---

check_id is for deduplication which is currently not working from Vero's side. We've sent a mail regarding this and Vero acknoledged the same. We're still waiting for the fix.
Using sourceFromGenericMap is not suggested because there might be a case where originalTimestamp and timestamp are same, which will not let the deduplication work as expected.
We'll update the "sourceKeys": "timestamp" to "sourceKeys": "originalTimestamp" as originalTimestamp is more consistent.